Unity/Assets/Scripts/HybidNative.cs

```diff
@@ -0,0 +1,45 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+using UnityEngine.Android;
+
+public class HybidNative : MonoBehaviour
+{
+
+    private HyBidBanner banner;
+    private bool isBannerLoading;
+
+	public string appToken;
+	public string placement;
+
+    [SerializeField]
+	private Button _buttonLoadTop;
+
+    [SerializeField]
+	private Button _buttonLoadBottom;
+
+    void Start()
+    {
+        banner = HyBidAdViewFactory.createHyBidAdView(this);
```

Comment:
will all your changes break the iOS-ready functionality? Have you checked that?

Reply:
I'm creating a merge branch to fix these conflicts. I'll mix the commits and then merge into the PR branch to solve the issues

Reply:
the thing is there some point which might not cause conflicts. For example, listeners removal will not cause any conflict, I believe, but can break the functionality

Reply:
Yes for sure. I meant conflicts in the functionality in this case. But no worries I'll mix it without breaking the functionality.